code/game/machinery/cryopod.dm

```diff
@@ -768,9 +768,13 @@
 
 	return ..()
 
+
 /proc/cryo_ssd(var/mob/living/carbon/person_to_cryo)
 	if(istype(person_to_cryo.loc, /obj/machinery/cryopod))
 		return 0
+	if(istype(person_to_cryo.loc, /obj))
```

Comment:
We have an `isobj(...)` define to do this. I'm not sure if that's preferred or not, so up to you if you want to keep it like this or use the define.